src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -829,13 +829,13 @@ public enum LineWrapMode
         /// Hyphenation mode will move part of the word (at possible hyphen locations)
         /// to the next line and draw a hyphen at the end of the line.
```

Comment:
We don't use the future tense for API description except for special cases.
Could you all descriptions as follow?

```suggestion
        /// Hyphenation mode moves part of the word (at possible hyphen locations) 
        /// to the next line and draws a hyphen at the end of the line.
```